frontends/qt/main.cpp

```diff
@@ -54,6 +54,15 @@ int main(int argc, char *argv[])
 //     qputenv("QT_AUTO_SCREEN_SCALE_FACTOR", QByteArray("1"));
 // #endif // QT_VERSION
 
+// QT configuration parameters which change the attack surface for memory
+// corruption vulnerabilities
+#if QT_VERSION >= QT_VERSION_CHECK(5,8,0)
```

Comment:
Safer to produce an error if the version is < 5.8?

Reply:
If we want to enable those protections, then I guess it would be good to not accidentally skip them by using an old Qt version.

Is there a reason to support older versions?

Reply:
It seems preferable to ship a newer version of QT when we build and support any specific release. What version should we target? QT 5.11?

Reply:
> Is there a reason to support older versions?

On Debian Stable, 5.7 is the current QT library version. Ideally, a user should be able to build with the correct system libraries. This will also aid us in building smaller, easier to manage packages.

Reply:
I would always target Long Term Support releases, which is currently Qt 5.9.x .